.tmp-empty-package/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "empty-package",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT"
+}
```

Comment:
Was this mistakenly committed?

Reply:
Unfortunately not. Styled components have bug in [their typings.](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33311). And it's workaround to fix it. It'll be removed after full migration to CSS modules

Reply:
let's ccreate a ticket with checkboxes to fix after merging this, please

Reply:
I save all such comments to my list and will add as description in the next PR